rocketpy/supplement.py

```diff
@@ -96,10 +116,30 @@ def filled_volume(self, volume):
             self._filled_volume = 0
             self._filled_height = 0
 
+    @property
+    def empty_volume(self):
+        return self.volume - self._filled_volume
+
+    @property
+    def empty_centroid(self):
+        if self.empty_volume == 0:
+            return 0
```

Comment:
I know this will not change any results, but shouldn't it return `self.height` if we were to consider the limit of the centroid poisition as the empty volume becomes zero?

Reply:
Yes, I believe you are correct. As you said it does not affect mathematically the results since this only happens when `empty_volume = 0`.

That if clause likely was not needed, it was more of a extra check so that nothing weird happens when the tank is completely filled.